terraform/aws-accounts/cloud-platform-aws/account/opensearch.tf

```diff
@@ -149,6 +176,16 @@ resource "aws_opensearch_domain" "live_modsec_audit" {
   }
 }
 
+# add vanity url to cluster 
+resource "aws_route53_record" "opensearch_custom_domain" {
+  zone_id = data.aws_route53_zone.et_cloud_platform_justice_gov_uk.zone_id
+  name    = "et-logs"
```

Comment:
I think this should be `logs` for the prod account

Reply:
good catch